doc/version_history.rst

```diff
@@ -11,16 +11,20 @@ v1.6.0
 
 Changes:
 
-* Add rule `rules.DewPointDepression`.
-  This rule requires ts_xml 10.1.
-* Add rule `rules.Humidity`.
-* Add rule `rules.MTCCWFollowingRotator`: warn when the MT camera cable wrap is not following the camera rotator.
-* Add classes  `FieldWrapperList`, `BaseFilteredFieldWrapper`, `FilteredFieldWrapper`, and `IndexedFilteredFieldWrapper`.
+* Add rules (most of which require ts_xml 10.0):
```

Comment:
In line 49 you put `ts_xml 10.1`. Which one is it?

Reply:
It's 10.1. Thank you for catching that.